src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -655,7 +680,11 @@ internal UndoRedoRecorder UndoRecorder
             get { return undoRecorder; }
         }
 
+        /// <summary>
+        /// Responsible for resolving a partial class name to its fully resolved name
```

Comment:
What this returns? An instance of ElemenetResolve? then say Returns ElementResolver. This property resolves partial class name to fully resolved name.